src/cascadia/TerminalApp/userDefaults.json

```
@@ -31,10 +45,28 @@
         ]
     },
 
-    // Add custom color schemes to this array
+    // Add custom color schemes to this array.
+    // To learn more about color schemes, visit https://aka.ms/terminal-color-schemes
     "schemes": [],
 
-    // Add any keybinding overrides to this array.
-    // To unbind a default keybinding, set the command to "unbound"
-    "keybindings": []
+    // Add custom keybindings to this array.
+    // To unbind a key combination from your defaults.json, set the command to "unbound".
+    // To learn more about keybindings, visit https://aka.ms/terminal-keybindings
+    "keybindings":
+    [
+        // Copy and paste are bound to Ctrl+Shift+C and Ctrl+Shift+V in your defaults.json.
+        // These two lines additionally bind them to Ctrl+C and Ctrl+V.
+        // To learn more about selection, visit https://aka.ms/terminal-selection
+        { "command": "copy", "keys": "ctrl+c" },

```

```suggestion
        { "command": {"action": "copy", "singleLine": false }, "keys": "ctrl+c" },
```
Thoughts? It would add discoverability.